Userland/Applications/Spreadsheet/SpreadsheetWidget.cpp

```diff
@@ -495,20 +495,70 @@ void SpreadsheetWidget::change_cell_static_color_format(Spreadsheet::FormatType
 {
     VERIFY(current_worksheet_if_available());
 
-    auto dialog = GUI::ColorPicker::construct(Color::White, window(), "Select Color");
-    if (dialog->exec() == GUI::Dialog::ExecResult::OK) {
+    auto preview_color_in_selected_cells = [this, format_type](Gfx::Color color) {
```

Comment:
I'll let @alimpfard review this file 😅

Reply:
Yeah... It turned into a bit of lambda spaghetti :sweat_smile:

Reply:
Lambda soup, but sure, LGTM.